src/main/java/org/folio/roles/domain/entity/RoleEntity.java

```diff
@@ -37,4 +42,12 @@ public class RoleEntity extends Auditable {
    */
   @Column(name = "description", nullable = false)
   private String description;
+
+  /**
+   * The source of the role.
+   */
+  @Enumerated(EnumType.STRING)
+  @JdbcTypeCode(SqlTypes.NAMED_ENUM)
+  @Column(name = "source", columnDefinition = "source_type", updatable = false)
+  private SourceType source;
```

Comment:
there is specialization of `RoleEntity` called `LoadableRoleEntity`. it already has a field to classify roles: type
it'd consider moving type field to `RoleEntity`, because having two fields for similar purpose looks too excessive

Reply:
According to US's description the goal is _"to prevent local administrators from editing consortium-managed policies"_.. `type` field in `LoadableRoleEntity` is used for the same purpose: if you look at `ApiRoleCapabilityService` or `ApiRoleCapabilitySetService` you'll see that it forbids modification of `DEFAULT` roles.

Reply:
Removed all modifications to Role entity and only focus on the Policy source field

Reply:
@dmtkachenko As the linked stories suggest, `source` is needed to also differentiate roles and policies created in consortium or member tenants, thus having the CONSORTIUM and USER values. I suggest replacing the loadableRoleType with the new source field as well and also handle importing default reference data as SYSTEM roles.

Reply:
After some considerations, replacing `loadableRoleType` would require some risky changes, so I propose to extend it with an additional value to differentiate `USER` and `CONSORTIUM` roles and moving the type to `RoleEntity`. Let me know your opinions on this @OleksiiKuzminov @dmtkachenko

Reply:
Hi @dmtkachenko , @OleksiiKuzminov 
Based on this conversation here is mapping:
DEFAULT - mapped to our SYSTEM, and system role could not be deleted, updated
SUPPORT - mapped to our USER , and any actions can be applied
CONSORTIUM - new value in EntityLoadableRoleType, could not be deleted, updated.
So just need to add 1 new value CONSORTIUM into EntityLoadableRoleType, do you agree with this approach?